### PR TITLE
Map a bunch of constants in Entity and MobEntity

### PR DESCRIPTION
**Pull request by Sollace**
I set out to do a remapping of the Goals classes, but ended up doing this instead.

For any inlined constants I've also included a reference to a method where you can see that constant being used. Names were in some cases determined by deductive observation: 
1. Constants dealing with similar or related functions (such as mob's armour) would be grouped together, have similar types, similar visibility levels, or have usages in close proximity to each other.
2. Constants added since 1.17 are most likely to relate to known changes in 1.18 snapshots (i.e. Player sprinting into walls no longer stopping sprinting) and could be checked in reverse by visiting expected locations.

Based on these, I was able to look into code surrounding or related to constant that were more easily identified for usages of more difficult constants and to rule out other possible uses that are deemed unlikely due to being unrelated to the core purpose of the set of constant's I'm attempting to map. 

`FREEZING_DAMAGE_INTERVAL`, for instance, is located very close to `DEFAULT_MIN_FREEZE_DAMAGE_TICKS`. The latter is the same return value as returned by `getMinFreezeDamageTicks` and the former is used in the same place that `getMinFreezeDamageTicks` is being called (in `LivingEntity#tickMovement`.

### CODE REVIEW

File: mappings/net/minecraft/entity/Entity.mapping [hunk 1]
```diff
@@ -12,21 +12,50 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
...
 	FIELD field_26995 removalReason Lnet/minecraft/class_1297$class_5529;
 	FIELD field_26996 entityChangeListener Lnet/minecraft/class_5569;
+	FIELD field_26997 lastChimeIntensity F
 	FIELD field_27857 inPowderSnow Z
 	FIELD field_27858 FROZEN_TICKS Lnet/minecraft/class_2940;
+	FIELD field_28627 speed F
 	FIELD field_28628 wasInPowderSnow Z
 	FIELD field_28629 wasOnFire Z
+	FIELD field_29973 DEFAULT_FRICTION F
+		COMMENT The factor by which and entity's speed is reduced every tick.
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
		COMMENT The factor by which an entity's speed is reduced every tick.
```

File: mappings/net/minecraft/entity/mob/MobEntity.mapping [hunk 2]
```diff
@@ -2,13 +2,36 @@ CLASS net/minecraft/class_1308 net/minecraft/entity/mob/MobEntity
 	FIELD field_18074 positionTarget Lnet/minecraft/class_2338;
 	FIELD field_18075 positionTargetRange F
 	FIELD field_18279 holdingEntityId I
+	FIELD field_30083 DEFAULT_CAN_PICKUP_LOOT_CHANCE F
+		COMMENT Used by Zombies to control the chance that they spawn with the ability to pick up loot.
+		COMMENT
+		COMMENT @see ZombieEntity#initialize
+	FIELD field_30084 BASE_ENCHANTED_ARMOR_CHANCE F
+		COMMENT The base chance (before applying difficulty) that a mob's equiped armor can become enchanted.
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/mob/MobEntity.mapping]:
```suggestion
		COMMENT The base chance (before applying difficulty) that a mob's equipped armor can become enchanted.
```

File: mappings/net/minecraft/entity/mob/MobEntity.mapping [hunk 3]
```diff
@@ -2,13 +2,36 @@ CLASS net/minecraft/class_1308 net/minecraft/entity/mob/MobEntity
...
+		COMMENT
+		COMMENT @see MobEntity#enchantEquipment
+	FIELD field_30085 BASE_ENCHANTED_MAIN_HAND_EQUIPMENT_CHANCE F
+		COMMENT The base chance (before applying difficulty) that a mob's equiped item can become enchanted.
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/mob/MobEntity.mapping]:
```suggestion
		COMMENT The base chance (before applying difficulty) that a mob's equipped item can become enchanted.
```

**Review by Shnupbups — COMMENTED** [review]
just a couple spelling fixes

File: mappings/net/minecraft/entity/Entity.mapping [hunk 4]
```diff
@@ -12,21 +12,50 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
...
+	FIELD field_26997 lastChimeIntensity F
 	FIELD field_27857 inPowderSnow Z
 	FIELD field_27858 FROZEN_TICKS Lnet/minecraft/class_2940;
+	FIELD field_28627 speed F
 	FIELD field_28628 wasInPowderSnow Z
 	FIELD field_28629 wasOnFire Z
+	FIELD field_29973 DEFAULT_FRICTION F
+		COMMENT The factor by which an entity's speed is reduced every tick.
+		COMMENT <p>
+		COMMENT For example: horizontalSpeed = velocity.horizontalSpeed() * FRICTION_RATE
```
Comment (enbrain) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
		COMMENT For example: {@code horizontalSpeed = velocity.horizontalSpeed() * FRICTION_RATE}
```

File: mappings/net/minecraft/entity/Entity.mapping [hunk 5]
```diff
@@ -12,21 +12,50 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
...
+		COMMENT @see Entity#getVelocityAffectingPos
+	FIELD field_29992 DEFAULT_MIN_FREEZE_DAMAGE_TICKS I
+		COMMENT @see Entity#getMinFreezeDamageTicks
+	FIELD field_29993 FREEZING_DAMAGE_INTERVAL I
+		COMMENT @see LivingEntity#tickMovement
 	FIELD field_29994 UUID_KEY Ljava/lang/String;
 	FIELD field_33758 hasVisualFire Z
+	FIELD field_34927 collidedSoftly Z
+		COMMENT Whether the collision velocity (speed at which the entity hit a given surface)
+		COMMENT is greated than {@link Entity#MAX_SOFT_COLLISION_SPEED}
```
Comment (enbrain) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
		COMMENT is lower than {@link Entity#MAX_SOFT_COLLISION_SPEED}.
```

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/entity/Entity.mapping [hunk 6]
```diff
@@ -12,21 +12,50 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
...
+	FIELD field_34928 MAX_SOFT_COLLISSION_SPEED F
```
Comment (YanisBft) [inline mappings/net/minecraft/entity/Entity.mapping]:
```suggestion
	FIELD field_34928 MAX_SOFT_COLLISION_SPEED F
```

**Review by YanisBft — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

**Pull request merged**